storage-proofs/src/stacked/params.rs

```diff
@@ -361,6 +479,71 @@ impl<H: Hasher> Labels<H> {
         self.labels.is_empty()
     }
 
+    pub fn labels_for_layer(&self, layer: usize) -> DiskStore<H::Domain> {
+        assert!(layer != 0, "Layer cannot be 0");
+        assert!(
+            layer <= self.layers(),
+            "Layer {} is not available (only {} layers available)",
+            layer,
+            self.layers()
+        );
+
+        let row_index = layer - 1;
+        let config = self.labels[row_index].clone();
+        assert!(config.size.is_some());
+
+        DiskStore::new_from_disk(config.size.unwrap(), &config).unwrap()
+    }
+
+    /// Returns label for the last layer.
+    pub fn labels_for_last_layer(&self) -> DiskStore<H::Domain> {
+        self.labels_for_layer(self.labels.len() - 1)
+    }
+
+    /// How many layers are available.
+    fn layers(&self) -> usize {
+        self.labels.len()
+    }
+
+    /// Build the column for the given node.
+    pub fn column(&self, node: u32) -> Result<Column<H>> {
+        let rows = self
+            .labels
+            .iter()
+            .map(|label| {
+                assert!(label.size.is_some());
+                let store = DiskStore::new_from_disk(label.size.unwrap(), &label).unwrap();
```

Comment:
SUGGESTION: Given that you're returning a `Result<Column<H>>`, I'd suspect that it's not rocket science to remove the `unwrap` and instead short-circuit the map if `DiskStore::new_from_disk` produces an error.